### PR TITLE
HM-2773: Admin reports

### PR DESCRIPTION
**Pull request by Revit**

### CODE REVIEW

File: src/bundles/ApplicationsAdmin/components/ActivityReports/ActivityReports.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    })
+  }
+
+  render() {
+      return (
+        <form onSubmit={this.handleSubmit}>
+        <AUheading size="xl" level="1">
+          Activity reports
+        </AUheading>
+        <p>Download opportunity report data for your agency or a current list of approved sellers.</p>
```
Comment (tobiasbrown) [inline src/bundles/ApplicationsAdmin/components/ActivityReports/ActivityReports.js:37]: I don't think we need the same description as buyer reports.
```suggestion
```

File: src/bundles/ApplicationsAdmin/components/ActivityReports/ActivityReports.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        <input type="radio" id="allSellersCategoryMaxDailyRates" name="reportType" value="allSellersCategoryMaxDailyRates" onChange={this.handleChange}/>
+        <label htmlFor="allSellersCategoryMaxDailyRates">All Sellers Category Max Daily Rates</label>
```
Comment (tobiasbrown) [inline src/bundles/ApplicationsAdmin/components/ActivityReports/ActivityReports.js:39]:
```suggestion
        <label htmlFor="allSellersCategoryMaxDailyRates">All sellers category max daily rates</label>
```

File: src/bundles/ApplicationsAdmin/components/ActivityReports/ActivityReports.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        <input type="radio" id="allSellersApprovedInCategory" name="reportType" value="allSellersApprovedInCategory" onChange={this.handleChange}/>
+        <label htmlFor="allSellersApprovedInCategory">All Sellers Approved in a Category</label>
```
Comment (tobiasbrown) [inline src/bundles/ApplicationsAdmin/components/ActivityReports/ActivityReports.js:41]:
```suggestion
        <label htmlFor="allSellersApprovedInCategory">All sellers approved in a category</label>
```

File: src/bundles/ApplicationsAdmin/components/ActivityReports/ActivityReports.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        <AUbutton type="submit" disabled={!this.state.acceptEnabled}>
+          Download reports
+        </AUbutton>
```
[the remark this answers is not in the document]
Comment (Revit) [reply src/bundles/ApplicationsAdmin/components/ActivityReports/ActivityReports.js]: indented

**Review by dta-tom — APPROVED** [review]
Looks fine

**Pull request merged**